app/src/main/res/layout/item_message_thread.xml

```diff
@@ -1,68 +1,87 @@
+        android:layout_marginLeft="12dp"
+        android:layout_marginRight="12dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/thread_lbl_preview"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
-    <LinearLayout
-        android:layout_width="match_parent"
+    <TextView
+        android:id="@+id/thread_lbl_participants"
+        style="@style/TextAppearance.AppCompat.Title"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:paddingLeft="7dp"
-        android:paddingStart="7dp"
-        android:orientation="vertical">
-
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="horizontal">
+        android:ellipsize="middle"
+        android:fontFamily="sans-serif-medium"
+        android:paddingEnd="4dp"
+        android:paddingLeft="0dp"
+        android:paddingRight="4dp"
+        android:paddingStart="0dp"
+        android:singleLine="true"
+        android:textSize="17sp"
+        app:layout_constraintBottom_toTopOf="@+id/thread_lbl_subject"
+        app:layout_constraintEnd_toStartOf="@+id/thread_lbl_last_updated"
+        app:layout_constraintStart_toEndOf="@+id/thread_icon"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="John Snow" />
 
-            <TextView
-                android:id="@+id/thread_lbl_participants"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_weight="1"
-                style="@style/WsText.subHead"
-                android:singleLine="true"
-                android:ellipsize="end"
-                tools:textStyle="bold"
-                tools:text="John Snow"/>
+    <TextView
+        android:id="@+id/thread_lbl_last_updated"
+        style="@style/WsText.light"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:gravity="center_vertical"
+        android:singleLine="true"
+        app:layout_constraintBottom_toTopOf="@+id/thread_lbl_subject"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/thread_lbl_participants"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="1d" />
 
-            <TextView
-                android:id="@+id/thread_lbl_last_updated"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:gravity="center_vertical|end"
-                style="@style/WsText.light"
-                android:singleLine="true"
-                tools:text="1d"/>
-        </LinearLayout>
+    <TextView
+        android:id="@+id/thread_lbl_subject"
```

Comment:
The padding between the participants, subject and preview seems a bit too big. What do you think? Maybe we can make the participants bold instead of using three different font sizes?